### ISSUE TITLE
`/corpus_config`: Undocumented features

### ISSUE THREAD

**Issue by janiemi** [spraakbanken/korp-backend#13]
When making some modifications to function `get_mode` called by `/corpus_config`, I noticed two features that appear currently undocumented:

1. When you specify `corpora`, a corpus id may be prefixed by a path to the configuration file (subdirectories under `CORPUS_CONFIG_DIR/corpora`): https://github.com/spraakbanken/korp-backend/blob/229b2744d450976704718744e4eee1819b125986/korp.py#L3441-L3444
2. A corpus attribute definition may use an attribute preset but amend it or override some of its properties by specifying a mapping (`dict`) value with the value of `preset` specifying the name of the preset to whose content the rest of the values of the mapping are added. For exapmle:
   ```yaml
   pos_attributes:
     - lemma:
         preset: lemma
         stringify: lemma
   ```
   The code is here: https://github.com/spraakbanken/korp-backend/blob/229b2744d450976704718744e4eee1819b125986/korp.py#L3485-L3491 and here: https://github.com/spraakbanken/korp-backend/blob/229b2744d450976704718744e4eee1819b125986/korp.py#L3517-L3520

Are these features undocumented because of being experimental or obsolete, or has their documentation been omitted because they are (currently) not used in Språkbanken’s corpus configuration?

**Comment by MartinHammarstedt** [comment]
1. This "hack" was added to be able to separate the config files for user generated Mink corpora from our regular corpora, but then we ended up running a completely separate Korp instance for Mink instead. It's currently not supported by the front end, so I might just remove the feature completely (and then add it back later if it's needed).
2. The `preset:` feature should definitely be documented. Thanks for pointing it out!
